src/handlers/processors.ts

```diff
@@ -54,7 +54,7 @@ export const processors: Record<string, Handler> = {
   [GithubEvent.ISSUES_CLOSED]: {
     pre: [nullHandler],
     action: [issueClosedCallback],
-    post: [incentivizeCreatorComment, incentivizeComments],
+    post: [incentivizeCreatorComment, incentivizeComments, incentivizePullRequestReviews],
```

Comment:
@0xcodercrane is there a way to pass in data to each? We need this to calculate and consolidate payment permit amounts per contributor.

Reply:
I think we can create a high-level method like `calculateIncentives`  and put 3 methods in it sequentially.

Reply:
let me consolidate them in a one method.

Reply:
> let me consolidate them in an one method.

How is this looking?

Reply:
https://github.com/ubiquity/ubiquibot/pull/714

I think after merging this, we can handle it on https://github.com/ubiquity/ubiquibot/pull/714